### PR TITLE
Fixing table stream visivility

### LINKED ISSUE #1300: Windows linker error table_stream_utility.h
I get an error when compiling the current master branch in windows:

image

Does anyone know how to solve this?

Thanks!

### PR DESCRIPTION
**Pull request by roigcarlo**
Fixes #1300

### CODE REVIEW

File: kratos/utilities/table_stream_utility.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ namespace Kratos
 ///@name  Functions
 ///@{
     
-class KRATOS_API(KRATOS_CORE) TableStreamUtility
+class TableStreamUtility
```
Comment (loumalouomega) [inline kratos/utilities/table_stream_utility.h:43]: Thanks, but last time when we had problems with Windows compilation  the solution was precisely to add this line

**Review by loumalouomega — DISMISSED** [review]
Thanks

File: kratos/utilities/table_stream_utility.h (hunk 1 above)
Comment (roigcarlo) [reply kratos/utilities/table_stream_utility.h:43]: I will try to compile this in windows, but pretty sure that was never working in windows in the first place.

**Comment by roigcarlo** [comment]
Indeed the same thing was happening with the `TableStream`. Now it compiles in win, can you test that it works fine?

**Review by pooyan-dadvand — APPROVED** [review]
:+1: 

I assume that you have run the tests

**Pull request merged**